server/applications.json

```diff
@@ -656,8 +656,10 @@
                         "windows": [
                             "C:\\Program Files\\Side Effects Software\\Houdini 19.5.805\\bin\\houdini.exe"
                         ],
-                        "darwin": [],
-                        "linux": []
+                        "darwin": [
+                            "/Applications/Houdini/Houdini19.5.805/Frameworks/Houdini.framework/Versions/Current/Resources/bin/houdini"
+                        ],
+                        "linux": ["/opt/houdini19.5.805/bin/houdini"]
```

Comment:
@Lypsolon any chance you can confirm this is e.g. matching what would be default path for houdini in your linux machine?